### DISCUSSION TITLE
Cannot use connectRefinementList  as it is not under widget

### DISCUSSION THREAD

**Discussion by Piusha** [algolia/instantsearch#4529] [General] [answered]
What is the way If I want to create custom refinement list widget with my own UI. Earlier we had `connectRefinementList` function but in latest instantsearch NPM version does not available `connectRefinementList` 

Is there any way we can use it ?
When I checked under connectors in instantsearch package  i can see connectRefinementList is there. But cannot use it as we used in earlier versions

Thanks
![Screenshot from 2020-10-09 12-52-06](https://user-images.githubusercontent.com/8476584/95554948-aea8bf00-0a2e-11eb-8219-b9aa0d34f310.png)

**Comment by Haroenv** [comment, marked as answer]
Can you share the code you're using? Here's an example on how to use connectRefinementList https://codesandbox.io/s/bold-wildflower-9o1ng?file=/src/app.js via UMD

Via npm it's like this: https://codesandbox.io/s/inspiring-flower-1nqgo?file=/src/app.js

both seem to work just fine for me, but to help you properly, please try recreating your error in a sandbox like this

**Reply by Piusha** [reply]
@Haroenv 
Thank you for the response,
If you can check this and let me know what is the issue in here
https://codesandbox.io/s/happy-rosalind-j8yix?file=/src/app.js

**Reply by Piusha** [reply]
Hey Thank for the Time, I think I know the issue

**Reply by Haroenv** [reply]
all right, glad you have a working experience again now @Piusha !
